Modules/MUON/MCH/src/PedestalsTask.cxx

```diff
@@ -212,42 +145,36 @@ void PedestalsTask::fill_noise_distributions()
     for (int bi = 0; bi < ih->second->GetXaxis()->GetNbins(); bi++) {
       for (int ci = 0; ci < ih->second->GetYaxis()->GetNbins(); ci++) {
         float noise = ih->second->GetBinContent(bi + 1, ci + 1);
-        if (noise < 0.001)
+        if (noise < 0.001) {
```

Comment:
Can't we have configurable thresholds instead of hard-coded ones ? (here and in L245~ below)

Reply:
This is actually just to skip channels with RMS=0 when filling the 1-D noise distributions, so that one avoids a prominent peak at zero.